src/screens/dialogs/upload-file-dialog.tsx

```diff
@@ -0,0 +1,82 @@
+import { useContext } from "react";
+import toast from "react-hot-toast";
+import { uploadfileService } from "../../services/files/upload-file.service";
+import { Dialog } from "../../components/Dialog";
+import {
+  FilesDialogsContext,
+  AuthContext,
+  AVAILABLE_DIALOGS,
+  FilesContext
+} from "../../context/index";
+
+interface INewFile {
+  isFile: boolean;
+  isReady: boolean;
+  name: string;
+  size: number;
+  uuid: string;
+}
+
+export const UploadFileDialog = () => {
+  // Dialog state
+  const { closeDialog, dialogsVisibilityState } =
+    useContext(FilesDialogsContext);
+  const isOpen = dialogsVisibilityState[AVAILABLE_DIALOGS.UPLOAD_FILE];
+  const { addFile, currentDirectory } = useContext(FilesContext);
+  const { session } = useContext(AuthContext);
+
+  const uploadFile = async () => {
+    const fileInput = document.querySelector<HTMLInputElement>("#myfile");
```

Comment:
I think it would be better if you use an `useRef` or an `state` to store the files instead of selecting the input with the `querySelector` method.